thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/FetchMetricDataAndExistingAnomaliesTool.java

```diff
@@ -0,0 +1,327 @@
+package com.linkedin.thirdeye.tools;
+
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+import com.linkedin.thirdeye.api.DimensionMap;
+import com.linkedin.thirdeye.constant.AnomalyFeedbackType;
+import com.linkedin.thirdeye.datalayer.bao.AnomalyFunctionManager;
+import com.linkedin.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.bao.RawAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFeedbackDTO;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import com.linkedin.thirdeye.datalayer.dto.RawAnomalyResultDTO;
+import com.linkedin.thirdeye.datalayer.util.DaoProviderUtil;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import javax.xml.transform.Result;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+
+public class FetchMetricDataAndExistingAnomaliesTool {
+  private AnomalyFunctionManager anomalyFunctionDAO;
+  private MergedAnomalyResultManager mergedAnomalyResultDAO;
+  private RawAnomalyResultManager rawAnomalyResultDAO;
+
+  public FetchMetricDataAndExistingAnomaliesTool(File persistenceFile) throws Exception{
+    init(persistenceFile);
+  }
+
+  // Private class for storing and sorting results
+  public class ResultNode implements Comparable<ResultNode>{
+    long functionId;
+    String functionName;
+    private String filters;
+    DimensionMap dimensions;
+    DateTime startTime;
+    DateTime endTime;
+    double severity;
+    AnomalyFeedbackType _feedbackType;
```

Comment:
Maintain the same convention in field names, feedbackType instead of _feedbackType